decidim-core/config/locales/en.yml

```diff
@@ -361,6 +361,7 @@ en:
       document:
         download: Download file
       documents:
+        component_documents: "%{name} Documents"
```

Comment:
Downcase "Documents"? So the string would be "Meetings documents"

Reply:
You are right